lib/runner.js

```diff
@@ -529,7 +533,13 @@ Runner.prototype.runTests = function (suite, fn) {
     }
 
     if (test.isPending()) {
-      self.emit('pending', test);
+      if (self.forbidPending) {
+        test.isPending = alwaysFalse;
```

Comment:
A pending test is a failure, because a pending test (a synchronous one, anyway) actually throws an exception.  We catch that in the `Runnable`, then set the pending flag, then continue on to the "fail" handler in `Runner`.  At least, if memory serves...